src/xdocs/checks/misc/newlineatendoffile.xml.template

```diff
@@ -94,51 +94,63 @@ StaticMethodCandidateCheck.name = Static Method Candidate
       </subsection>
 
       <subsection name="Examples" id="Examples">
-        <p id="Example1-config">
-          To configure the check:
-        </p>
-        <macro name="example">
-          <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/newlineatendoffile/Example1.txt"/>
-          <param name="type" value="config"/>
-        </macro>
-        <p id="Example1-code">Example:</p>
-        <macro name="example">
-          <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/newlineatendoffile/Example1.txt"/>
-          <param name="type" value="code"/>
-        </macro>
+          <p id="Example1-config">
```

Comment:
Please restore indentation of tag `p` and whole block.
This might be a reason of xml violation